packages/dbos-cloud/organizations/organization.ts

```diff
@@ -0,0 +1,74 @@
+import axios, { AxiosError } from "axios";
+import { isCloudAPIErrorResponse, handleAPIErrors, getCloudCredentials, getLogger } from "../cloudutils.js";
+
+
+export async function orgInvite(host: string) {
+    const logger = getLogger();
+    const userCredentials = await getCloudCredentials();
+    const bearerToken = "Bearer " + userCredentials.token;
+  
+    try {
+        const res = await axios.get(`https://${host}/v1alpha1/${userCredentials.organization}/organizations/secret`, {
+            headers: {
+              "Content-Type": "application/json",
+              Authorization: bearerToken,
+            },
+          });
+      
+      
+      logger.info(`Secret for inviting user to organization ${userCredentials.organization}: `);
+      logger.info(res.data);
```

Comment:
I suggest we also allow users to specify `--json` for this command: it will make it easier for them (and us) to write tests